prebuilt/libopenssl.gyp

```diff
@@ -117,7 +121,7 @@
 						},
 					],
 					[
-						'OS == "android"',
+						'toolset_os == "android"',
 						{
```

Comment:
There needs to be an additional 
```
 							'conditions':
 							[
 								[
 									'OS == "android"',
 									{
```
here like with libicu.gyp, otherwise the configure step complains about `<(android_subplatform)`